src/init.cpp

```diff
@@ -532,6 +532,7 @@ void SetupServerArgs(NodeContext& node)
 #else
     hidden_args.emplace_back("-logthreadnames");
 #endif
+    argsman.AddArg("-logfunctionnames", strprintf("Prepend debug output with name of the originating function (default: %u)", DEFAULT_LOGFUNCTIONNAMES), ArgsManager::ALLOW_ANY, OptionsCategory::DEBUG_TEST);
```

Comment:
nit: I thought `-logsourcelocation` was a better name

Reply:
Agreed! Now using `-logsourcelocations`. The plural suffix to be consistent with `-logthreadnames`.